lib/util/i18n.js

```diff
@@ -0,0 +1,31 @@
+import flatten from 'flat'
+
+/*
+ * Load the localized strings for all languages here,
+ * merge with the provided customized strings from state.otp.config.language
+ * (overrides can be defined per language or for all languages)
+ * and flatten to a map of
+ *   id => string.
+ * FIXME: Load languages on demand using fetch.
+ */
+export async function loadLocaleData (locale, customMessages) {
+  let messages
+  switch (locale) {
+    case 'fr-FR':
```

Comment:
This should be tolerant of incomplete (`fr` rather than just `fr-FR`) or miscapitalized locale strings. So perhaps a more appropriate approach for now would be to do something like:
```javascript
// Call getMatchingLocale in this method
const matchedLocale = getMatchingLocaleString(locale)
// Define locales and other method above this method.
const locales = {
  en: {
    us: 'en-US'
  },
  fr: {
    fr: 'fr-FR'
  }
}

function getMatchingLocaleString (locale = '', defaultLocale = 'en-US') {
  const [lang, region] = locale.toLowerCase().split('-')
  const language = locales[lang]
  if (!language) return defaultLocale
  const defaultRegion = Object.keys(language)[0]
  return language[region] || language[defaultRegion]
}
```

Reply:
Updated that code in this commit: https://github.com/opentripplanner/otp-react-redux/pull/375/commits/f5a74fabe0e292e3a3f6cb17278548d063945c14

Reply:
I disagree with @landonreed and don't this adds any value. The only way this makes sense is if raw user input is expected. Although this is how things are done with this very alpha branch of code, I fully do not expect users to be typing in raw locale-lang strings with more matured code. Please revert.

Reply:
There is truth to @evansiroky's comment above, as for each deployment, the user would eventually only choose from a predefined set of languages/locales. How about keeping the proposed code until we find out after adding the UI language selector whether it is still useful?

Reply:
I'd wager that it won't be useful at the point, but I'm fine with this proposal.